templates/simple/simple.cmd

```diff
@@ -87,6 +87,7 @@ use_oauth_services = {{group}}
 {% else %}
 +x509userproxy = "{{proxy}}"
 {% endif %}
+delegate_job_GSI_credentials_lifetime = 0
```

Comment:
Do we also need this in the various templates/dag*/*dag.sub files?

Reply:
Yes - but that will happen after #185 is done, if you're OK with that.